packages/survey-creator-core/src/property-grid/index.ts

```diff
@@ -1431,19 +1424,18 @@ export abstract class PropertyGridEditor implements IPropertyGridEditor {
     question: Question,
     options: ISurveyCreatorOptions,
     onClose?: (reason: "apply" | "cancel") => void
-  ): IPropertyEditorSetup {
+  ): void {
     const obj = (<any>question).obj;
     const surveyPropertyEditor = editor.createPropertyEditorSetup(
       obj,
       property,
       question,
       options
     );
-    if (!surveyPropertyEditor) return null;
+    if (!surveyPropertyEditor || !settings.showDialog) return;
```

Comment:
The setting seems undocumented: https://surveyjs.io/form-library/documentation/api-reference/settings